portal-api/src/v1/gef/controllers/application.controller.js

```diff
@@ -272,28 +263,17 @@ exports.changeStatus = async (req, res) => {
 
     // TODO: DTFS2-4705 - protect so that only a user with checker role and associated bank can submit to UKEF.
     if (status === DEAL_STATUS.SUBMITTED_TO_UKEF) {
-      await api.tfmDealSubmit(
-        dealId,
-        existingApplication.dealType,
-        req.user,
-      );
+      await api.tfmDealSubmit(dealId, existingApplication.dealType, req.user);
     }
   }
 
   // If status of correct type, send update email
-  if (
-    [
-      DEAL_STATUS.READY_FOR_APPROVAL,
-      DEAL_STATUS.CHANGES_REQUIRED,
-      DEAL_STATUS.SUBMITTED_TO_UKEF,
-    ].includes(status)
-  ) {
+  if ([DEAL_STATUS.READY_FOR_APPROVAL, DEAL_STATUS.CHANGES_REQUIRED, DEAL_STATUS.SUBMITTED_TO_UKEF].includes(status)) {
```

Comment:
Should we leave it in separate lines, as more than 3 should be in separate lines (if in future new statuses are added).